src/webgpu/shader/validation/shader_io/builtins.spec.ts

```diff
@@ -301,6 +335,8 @@ g.test('reuse_builtin_name')
   .beforeAllSubcases(t => {
```

Comment:
Just a note, since this test just use the built-in name in other contexts instead of actually using the built-in, this test should work for both device with/without related feature required. Maybe in the future we can test both cases.

Reply:
Done